examples/language-modeling/README.md

```diff
@@ -574,6 +574,46 @@ python3 ../gaudi_spawn.py --use_deepspeed  --world_size 8  run_lora_clm.py \
   --flash_attention_causal_mask True
 ```
 
+- Multi-card finetuning of Llama2-70B with DeepSpeed ZeRO-3 optimization, LoRA and FP8 precision:
```

Comment:
Maybe we can just modify existing command for FP8 precision (instead of adding a new one).

Reply:
I agree with Vivek

Reply:
Done. Updated the readme.